### ISSUE TITLE
Provide a category for the activities completing-read

### ISSUE THREAD

**Issue by jdtsmith** [alphapapa/activities.el#20]
This is what `marginalia` uses to add rich info annotations.  I could imagine annotations describing #windows, #files, #buffer, last modified file date, etc.  And of course a sort-function which puts activities with most-recently-modified files first.  

See the [docs for completing-read](https://www.gnu.org/software/emacs/manual/html_node/elisp/Programmed-Completion.html) for info on `metadata` `category`.

**Comment by alphapapa** [comment]
Yeah, and we'll probably need to ask @minad and @oantolin to chime in as well.

**Comment by alphapapa** [comment]
Since this may need coordination with Embark (or maybe not, I'm not sure), I'm deferring til 0.6.

**Comment by jdtsmith** [comment]
Marginalia actually recommends against depending on it:

> If you add your own completion commands to your package we recommend to specify an annotation-function or an affixation-function, avoiding the Marginalia dependency this way.

They recommend just making your own annotation function if you control the package. See #83 for a draft attempt.
